### PR TITLE
feature(gooddata-sdk): support date filters for get elements

### PR DESCRIPTION
**Pull request by mcermak**
JIRA: LX-161

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/workspace/content_service.py [hunk 1]
```diff
@@ -573,8 +575,8 @@ def get_label_elements(
             label_id (str):
                 Label ID. We support string or ObjId types.
                 String may not contain "label/" prefix, we append it if necessary.
-            depends_on (Optional[List[CatalogDependsOn]]):
-                Optional parameter specifying dependencies on other labels.
+            depends_on (Optional[List[CatalogDependsOnItem]]):
```
Comment (pcerny) [inline gooddata-sdk/gooddata_sdk/catalog/workspace/content_service.py:578]: Typo. `CatalogDependsOnItem` -> `DependsOnItem`

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by pcerny — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/343/ : FAILURE in 1m 34s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/493/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/497/ : SUCCESS in 6s

**Pull request merged**